src/registry.jl

```diff
@@ -44,12 +44,12 @@ add_format(format"SAS", UInt8[0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 add_format(format"Parquet", "PAR1", [".parquet"], [:ParquetFiles, LOAD])
 
 # Image formats
-add_format(format"PBMBinary", b"P4", ".pbm", [:ImageMagick])
-add_format(format"PGMBinary", b"P5", ".pgm", [:Netpbm])
-add_format(format"PPMBinary", b"P6", ".ppm", [:Netpbm])
-add_format(format"PBMText",   b"P1", ".pbm", [:ImageMagick, LOAD])
-add_format(format"PGMText",   b"P2", ".pgm", [:ImageMagick, LOAD])
-add_format(format"PPMText",   b"P3", ".ppm", [:ImageMagick, LOAD])
+add_format(format"PBMBinary", b"P4", ".pbm", [:ImageIO])
```

Comment:
Let's still keep ImageMagick, otherwise, it's a breaking change. Similar to other entries.

```suggestion
add_format(format"PBMBinary", b"P4", ".pbm", [:ImageIO], [:ImageMagick])
```

Reply:
The current version on master has the following entries:

- `P4 [:ImageMagick]`
- `P5 [:Netpbm]`
- `P6 [:Netpbm]`
- `P1 [:ImageMagick, LOAD]`
- `P2 [:ImageMagick, LOAD]`
- `P3 [:ImageMagick, LOAD]`

I can change it to

- `P4 [:ImageIO], [:Netpbm], [:ImageMagick]`
- `P5 [:ImageIO], [:Netpbm]`
- `P6 [:ImageIO], [:Netpbm]`
- `P1 [:ImageIO], [:Netpbm], [:ImageMagick, LOAD]`
- `P2 [:ImageIO], [:Netpbm], [:ImageMagick, LOAD]`
- `P3 [:ImageIO], [:Netpbm], [:ImageMagick, LOAD]`

Then the current options are available as fallback, and Netpbm can be used for all formats if ImageIO is not installed.

Reply:
I've added these changes.

Thanks @ianshmean and @johnnychen94 for your advice on getting this right.